src/components/TokenAdministrative.js

```diff
@@ -210,7 +216,7 @@ class TokenAdministrative extends React.Component {
         <div className='token-detail-bottom'>
           {renderBottom()}
         </div>
-        <HathorAlert ref="alertSuccess" text={this.state.successMessage} type="success" />
+        <HathorAlert ref={this.alertSuccessRef} text={this.state.successMessage} type="success" />
```

Comment:
String refs were deprecated, so I refactored `HathorAlert` to export a customized ref